### ISSUE TITLE
`set-output` and `save-state` are deprecated

### ISSUE THREAD

**Issue by muffl0n** [hashicorp/setup-terraform#252]
Since runner version 2.298.2 `set-output` and `save-state` are deprecated: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

```
The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

**Comment by dawrobel** [comment]
Have the same issue. Seems to be that terraform init|plan|apply triggering warnings once set from hashicorp/setup-terraform@v2 inside workflow

**Comment by magnetikonline** [comment]
Once tests are passing here and merged https://github.com/hashicorp/setup-terraform/pull/247 this warning will vanish. 

@actions/core 1.10.0 handles set-output using the new environment file mode - not outputting to STDOUT.

**Comment by piotrekkr** [comment]
We need to wait and it should be done soon. As a temp fix I used `terraform_wrapper: false` because I did not need this wrapper stuff

**Comment by muffl0n** [comment]
Fixed in #247, released in [v2.0.3](https://github.com/hashicorp/setup-terraform/releases/tag/v2.0.3). Thank you very much! ❤️

**Comment by kallangerard** [comment]
fixes #306
